CHANGELOG.md

```diff
@@ -22,14 +22,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- `tf-modules/cumulus` module now supports a `cmr_custom_host` variable that can be used to set to an arbitrary host for making CMR requests (e.g. `https://custom-cmr-host.com`).
+- `tf-modules/cumulus` module now supports a `cmr_custom_host` variable that can
+  be used to set to an arbitray  host for making CMR requests (e.g.
+  `https://custom-cmr-host.com`).
 - Added `buckets` variable to `tf-modules/archive`
 - **CUMULUS-2328**
   - Added `tf-modules/s3_credentials` module which contains resources to attach the `/s3-credentials` endpoint to an API gateway
 
+### Fixed
+
+- **CUMULUS-2394**
+  - Updated PDR and Granule writes to check the step function `workflow_start_time` against
+      the `createdAt` field  for each record to ensure old execution records do not
+      overwrite newer executions
```

Comment:
```suggestion
      the `createdAt` field  for each record to ensure old records do not
      overwrite newer ones
```

Nit to make it clearer that we're not talking about execution records

Reply:
👍🏻